metrics_utility/management/validation.py

```diff
@@ -54,10 +54,10 @@
     'main_indirectmanagednodeaudit',
     'total_workers_vcpu',
     'unified_jobs',
-    'job_host_summary_service',
     'main_jobevent_service',
     'execution_environments',
-    '',
```

Comment:
The empty string is there because otherwise validation rejects empty optional collectors.

(So, we need to keep it, or fix validation not to need it.)

Reply:
ah, I deleted it by accided. Nice catch!